integration_tests/tests/apply/risks_and_needs/health-needs/communication_and_language.cy.ts

```diff
@@ -69,7 +69,6 @@ context('Visit "communication and language" page', () => {
     CommunicationAndLanguage.visit(this.application)
     const page = new CommunicationAndLanguage(this.application)
 
-    page.describeAdditionalNeeds()
```

Comment:
Do we also need to remove something from an e2e test here?

Reply:
d'oh! yes we do

Reply:
fixed now, thanks